crates/client/src/pages/not_found.rs

```diff
@@ -0,0 +1,10 @@
+use yew::{function_component, html, Html};
```

Comment:
Creating new pages isn't bad, but I really don't know if it should be something what we should bother about now. Every subpage requires additional piece of work to make or integrate with another code like e.g. your disliked App. That doesn't improve our MVP, nor at this point we even don't know (or maybe only I) if that page should exists.

Reply:
Most websites have a 404 page so I think this should have one too.

Reply:
From one side 404 is nice, but in our situation (application) we probably can replace that by simple message box what will be popup on the user screen instead of creating whole new content. You can stay that, but I really don't see why we should care about this now, this sounds like one of last things which should be done :)